blade/procData/templates/csvUpload/csvUpload.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Upload CSV File</title>
+</head>
+<body>
+    <form method="post" enctype="multipart/form-data">
```

Comment:
later on we can style this a bit nicer using django-bootstrap, see https://www.youtube.com/watch?v=YmJ0TACm8oE

Reply:
I've been trying to install bootstrap but I'm getting this error.
<img width="985" alt="image" src="https://github.com/abaddonpuff/bladeART/assets/25719314/060c4f95-72e2-4e02-975e-df8289c02223">

Reply:
hm weird, not sure what the rich library has to do with it here, I am running pip 24.0 in my virtual env, what is your version? (`pip --version`)